k8s/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+namespace: coldfront
```

Comment:
This should be a different namespace right?

Reply:
Yup, I just force pushed the change.